### ISSUE TITLE
Can't fix json prettier error

### ISSUE THREAD

**Issue by zeusone** [Azure/azure-sdk-tools#6017]
I got alert of style https://github.com/Azure/azure-rest-api-specs/pull/23581/checks?check_run_id=12887313555, but when I tried to check and fix these errors, I got error.  I have followed the other issue: https://github.com/Azure/azure-rest-api-specs/issues/13186. But I was run this command under root folder.

PS C:\repos\hdinsight-new-version> npm run prettier-fix
npm WARN config global `--global`, `--local` are deprecated. Use `--location=global` instead.

> prettier-fix
> prettier --write specification/**/*.json

[error] Cannot find module 'C:\repos\hdinsight-new-version\scripts\prettier-swagger-plugin'
[error] Require stack:
[error] - C:\repos\hdinsight-new-version\node_modules\prettier\index.js
[error] - C:\repos\hdinsight-new-version\node_modules\prettier\cli.js
[error] - C:\repos\hdinsight-new-version\node_modules\prettier\bin-prettier.js

**Comment by Kai-lash** [comment]
I get the same error. I have followed this guidance too: https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/ci-fix.md#prettier-check

I tried these commands and get the same error again.
`npx prettier -c <filepath>`
`npx prettier -w <filepath>`

However, I was able to get past the issue by copying the file to be pretty-fied in the same directory.
So, from the repo root I ran this command, and it worked:
`npx prettier -w <**filename**>`

For some reason, the file path is not working for me.
